content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -0,0 +1,32 @@
+============
+Cycle Counts
+============
+
+In most companies, the stock is only counted once a year. That's why by default, after making an
+inventory adjustment, the scheduled date for the next count is set on the 31th of decemeber. But
+for some businesses it's crucial to have an accurate inventory count.
+
+The goal of cycle counts is to count more often at key locations to keep your critical stock levels
+accurate.
+
+Configuration
+=============
+
+In Odoo, cycle counts are location-based. That means that the frequency of the counts is defined on
+the storage location. That also means that to use cycle counts you need to activate the setting
+**Storage Locations**.
+
+.. image:: cycle_counts/storage-locations-setting.png
+   :align: center
+   :alt: Enable Storage Locations in Odoo settings.
+
+Then, go to the locations via :menuselection:`Inventory --> Configuration --> Locations`.
+
+Open a location and change the Inventory Frequency. In this example we want to
+count items in this loctation every 30 days. That means that once an adjustment
```

Comment:
- Replace the sentence on Lines 26-27 `That means that...30 days after.` with `Once the value is entered, click :guilabel:'Save' to apply the setting to the location. Now, once an inventory adjustment is applied to this location, the next scheduled count date will be automatically set based on the number of days in the :guilabel:'Inventory Frequency (Days)' setting.`